AppCenterDataStorage/AppCenterDataStorageTests/MSTokenExchangeTests.m

```diff
@@ -30,14 +30,13 @@
 static NSString *const cachedToken = @"mockCachedToken";
 static NSString *const kMSStorageReadOnlyDbTokenKey = @"MSStorageReadOnlyDbToken";
 static NSString *const kMSStorageUserDbTokenKey = @"MSStorageUserDbToken";
-static NSString *const MSDataStoreAppDocumentsPartition = @"readonly";
+static NSString *const kMSDataStoreAppDocumentsPartition = @"readonly";
```

Comment:
Why not use the public one in `MSDataStore.h`?

Reply:
Don't like re-using constants because it makes circular tests